prism-agent/service/server/src/main/scala/io/iohk/atala/credentialstatus/controller/CredentialStatusServiceEndpoints.scala

```diff
@@ -20,10 +21,18 @@ class CredentialStatusServiceEndpoints(
     authorizer: Authorizer[BaseEntity] // Will need for revocation endpoint
 ) {
 
+  private def obfuscateInternalServerError(e: ErrorResponse): ErrorResponse =
+    if e.status == StatusCode.InternalServerError.code then e.copy(detail = Some("Something went wrong"))
```

Comment:
I've been reading through the Error Handling ADRs and thinking about how best to give information to the end user - I understand why we may want to hide stack traces to prevent information disclosure (security) but can we provide more information to them to make it easier to work out what to do next?